docs/netlify/_redirects

```diff
@@ -0,0 +1,534 @@
+/docs/12.1/ManualRowResize.html /docs/12.1/javascript-data-grid/api/manual-row-resize/
+/docs/12.1/Matrix.html /docs/12.1/javascript-data-grid/api/matrix/
+/docs/12.1/Menu.html /docs/12.1/javascript-data-grid/api/menu/
+/docs/12.1/MergeCells.html /docs/12.1/javascript-data-grid/api/merge-cells/
+/docs/12.1/MergeCellsAction.html /docs/12.1/javascript-data-grid/api/merge-cells-action/
+/docs/12.1/MergedCellCoords.html /docs/12.1/javascript-data-grid/api/merged-cell-coords/
+/docs/12.1/MergedCellsCollection.html /docs/12.1/javascript-data-grid/api/merged-cells-collection/
+/docs/12.1/MultiColumnSorting.html /docs/12.1/javascript-data-grid/api/multi-column-sorting/
+/docs/12.1/MultipleSelectUI.html /docs/12.1/javascript-data-grid/multipleselectui
+/docs/12.1/NestedHeaders.html /docs/12.1/javascript-data-grid/api/nested-headers/
+/docs/12.1/NestedRows.html /docs/12.1/javascript-data-grid/api/nested-rows/
+/docs/12.1/NodeStructure.html /docs/12.1/javascript-data-grid/api/node-structure/
+/docs/12.1/ObserveChanges.html /docs/12.1/javascript-data-grid/api/observe-changes/
+/docs/12.1/OperatorsComponent.html /docs/12.1/javascript-data-grid/api/operators-component/
+/docs/12.1/Options.html /docs/12.1/javascript-data-grid/api/options/
+/docs/12.1/PersistentState.html /docs/12.1/javascript-data-grid/api/persistent-state/
+/docs/12.1/Queue.html /docs/12.1/javascript-data-grid/api/queue/
+/docs/12.1/RadioInputUI.html /docs/12.1/javascript-data-grid/radioinputui
+/docs/12.1/RecordTranslator.html /docs/12.1/javascript-data-grid/api/record-translator/
+/docs/12.1/RowsMapper.html /docs/12.1/javascript-data-grid/api/rows-mapper/
+/docs/12.1/SamplesGenerator.html /docs/12.1/javascript-data-grid/api/samples-generator/
+/docs/12.1/Search.html /docs/12.1/javascript-data-grid/api/search/
+/docs/12.1/Selection.html /docs/12.1/javascript-data-grid/api/selection/
+/docs/12.1/SelectionCalculations.html /docs/12.1/javascript-data-grid/api/selection-calculations/
+/docs/12.1/SelectionRange.html /docs/12.1/javascript-data-grid/api/selection-range/
+/docs/12.1/SelectUI.html /docs/12.1/javascript-data-grid/selectui
+/docs/12.1/Sheet.html /docs/12.1/javascript-data-grid/api/sheet/
+/docs/12.1/Stack.html /docs/12.1/javascript-data-grid/api/stack/
+/docs/12.1/Storage.html /docs/12.1/javascript-data-grid/api/storage/
+/docs/12.1/TableView.html /docs/12.1/javascript-data-grid/api/table-view/
+/docs/12.1/TextEditor.html /docs/12.1/javascript-data-grid/api/text-editor/
+/docs/12.1/Transformation.html /docs/12.1/javascript-data-grid/api/transformation/
+/docs/12.1/TrimRows.html /docs/12.1/javascript-data-grid/api/trim-rows/
+/docs/12.1/UndoRedo.html /docs/12.1/javascript-data-grid/api/undo-redo/
+/docs/12.1/UndoRedoSnapshot.html /docs/12.1/javascript-data-grid/api/undo-redo-snapshot/
+/docs/12.1/UnmergeCellsAction.html /docs/12.1/javascript-data-grid/api/unmerge-cells-action/
+/docs/12.1/ValueComponent.htm /docs/12.1/javascript-data-grid/valuecomponent.htm
```

Comment:
It looks like something got cut off while pasting the text. I'm not sure if it's just this line or if some other lines were cut.